### ISSUE TITLE
Parse error for empty input objects

### ISSUE THREAD

**Issue by kliuless** [rmosolgo/graphql-ruby#74]
```
mutation {
  createItem(input: 
    {
      clientMutationId: "112233",
      nested: {},  # <---- Empty input object can't be parsed
      title: "test mutation"
    }
  ) {
    returnField
  }
}
```
Error is: `#<GraphQL::ParseError: Extra input after last repetition at line 1 char 1.`
Please also verify that there can be spaces inside the braces.

**Comment by rmosolgo** [comment]
:grimacing:  Sorry! 

Fixed by d887f5d. I want to make a refactoring on `InterfaceType` today then push out a new version!

**Comment by kliuless** [comment]
Thanks for the quick reply. I finally grokked how the parse/transform code works, so I fixed the related transform code in PR #75 .
